### PR TITLE
Rollback attack

### PR DESCRIPTION
**Pull request by mcr**

### CODE REVIEW

File: opsawg-mud-acceptable-urls.md [hunk 1]
```diff
@@ -278,6 +278,19 @@ For a simple example, if the canonical MUD-URL is http://example.com/hello/there
 any URL that starts with http://example.com/hello/there/ would be acceptable, such as
 http://example.com/hello/there/revision2.json.
 
+One problem with these small changes is that malware could still express a
+MUD file that was previously valid, but which should no longer considered
+accurate.
+This is a rollback attack.
+This might result in the malware being able to reach destinations that turned
+out to be a mistake; a security fault.
+In order to combat, this, MUD managers SHOULD keep track of the list of
```
Comment (huitema) [inline opsawg-mud-acceptable-urls.md]: Extra comma? "In order to combat[,] this, MUD managers..."

File: opsawg-mud-acceptable-urls.md [hunk 2]
```diff
@@ -278,6 +278,19 @@ For a simple example, if the canonical MUD-URL is http://example.com/hello/there
...
+MUD-URLs that they have successfully retrieved, and if a device ever suggests a URL that was
+previously used, then the MUD manager should suspect that is a rollback attack.
+MUD managers are not typically not particularily constrained, and while the
+list of URLs could grow without bound, it is unlikely to be a burden.
```
Comment (huitema) [inline opsawg-mud-acceptable-urls.md:291]: "MUD managers are not typically resource constrained"?

**Review by huitema — APPROVED** [review]
2 nits, but looks good.

File: opsawg-mud-acceptable-urls.md (hunk 1 above)
Comment (mcr) [inline opsawg-mud-acceptable-urls.md]:
```suggestion
In order to combat this, MUD managers SHOULD keep track of the list of
```

File: opsawg-mud-acceptable-urls.md [hunk 3]
```diff
@@ -278,6 +278,19 @@ For a simple example, if the canonical MUD-URL is http://example.com/hello/there
...
+One problem with these small changes is that malware could still express a
+MUD file that was previously valid, but which should no longer considered
+accurate.
+This is a rollback attack.
+This might result in the malware being able to reach destinations that turned
+out to be a mistake; a security fault.
+In order to combat, this, MUD managers SHOULD keep track of the list of
+MUD-URLs that they have successfully retrieved, and if a device ever suggests a URL that was
+previously used, then the MUD manager should suspect that is a rollback attack.
+MUD managers are not typically not particularily constrained, and while the
```
Comment (mcr) [inline opsawg-mud-acceptable-urls.md]:
```suggestion
MUD managers are not typically resource constrained, and while the
```

**Pull request merged**